Mono.Collections.Generic/Collection.cs

```diff
@@ -99,13 +99,21 @@ public Collection ()
 			items = Empty<T>.Array;
 		}
 
+		static class EmptyArray<T> {
```

Comment:
We already have one of those, we're just not using it. It's used on line 99: `Empty<T>.Array`.

Reply:
oops, fixed.